### ISSUE TITLE
Add a README

### ISSUE THREAD

**Issue by dibyadas** [dibyadas/nn-digit-recognition#1]
Content can be:- 
`
A digit recognition app using a simple 3 layered neural network implemented purely in vanilla Javascript. The neural network has been pre-trained. It uses a input layer containing 784 (a neuron for each of 28x28 pixel image) neurons and an hidden layer and uses a softmax activation.
`
